cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -42,6 +42,7 @@ const (
 
 	defaultV18AdmissionControl = "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,ResourceQuota"
 	defaultV19AdmissionControl = "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,Priority,ResourceQuota"
+	defaultV19AdmissionControl1 = "Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,Priority,ResourceQuota"
```

Comment:
I would name this defaultV19AdmissionControl

Reply:
@luxas you mean deprecatedV19AdmissionControl

Reply:
@luxas and the above one deprecatedV19AdmissionControl

Reply:
yes